patcher.sh

```diff
@@ -59,12 +71,8 @@ patch_am2r ()
 
 	# Check for which OS we patch
 	if [ "$OSCHOICE" = "linux" ]; then
-
-		# Check whether Xdelta is installed
-		if [ !  -x "$(command -v xdelta3)" ] ; then
-			>&2 echo "Xdelta is not installed! Please install 'xdelta3' from your local package manager!"
-			exit 1
-		fi
+                checkInstalled "xdelta3"
```

Comment:
Considering xdelta is used in both Linux and Android, might also move it out to below the sed check.

Reply:
Sure :+1: